src/file-3.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:15380/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:15380/project/issues?id=sample&branch=feature2&issues=AWEyglU53ebqF5v7XZMM&open=AWEyglU53ebqF5v7XZMM)

[ISSUE_KEY]: AWEyglU53ebqF5v7XZMM